helm/tenant/templates/tenant-configuration.yaml

```diff
@@ -1,21 +1,10 @@
 {{- if (.Values.secrets) }}
-{{- print "# WARNING: '.secrets' is deprecated since v5.0.15 and will be removed in next minor release (i.e. v5.1.0). Please use '.tenant.configSecret' instead." }}
-{{- end }}
-{{- if and (.Values.secrets) (.Values.tenant.configSecret) }}
-{{- fail "ERROR: '.secrets' and '.tenant.configSecret' are mutually exclusive. Please use 'tenant.configSecret' instead." }}
-{{- end }}
-{{- if and (.Values.secrets) (not (.Values.secrets).existingSecret) }}
```

Comment:
This check for `existingSecret` is missing.

Reply:
See #2299.